prototype/cutlass/README.md

```diff
@@ -0,0 +1,3 @@
+# Cutlass Quant
+
+### Pythonic tools for defining `cutlass` kernels and quantization ops
```

Comment:
cc @andrewor14 who has also been thinking about CUTLASS in the context of https://github.com/pytorch-labs/ao/issues/86

Reply:
`Cutlass` is very neat.  

- `Cutlass 3.x` and the `CuTe` framework that it introduces has many useful primitives and patterns for defining bespoke kernels of relevance (mixed type GEMM, MoE, etc.), though it is targeted primarily at `sm90+` architectures.
- The `2.x` api has limited support for sub-byte mixed type quant kernels (without preprocessing weights to custom format -- I believe `pytorch` already has this integrated under `torch.quantization._quantized_conversions`).  

Currently working on using `Cutlass 3.x` / `CuTe` to adapt / improve `pre-Hopper` kernels useful for quant ops.  Would love to also test on `Hopper` but unfortunately don't have access to H100.